src/components/Main/main.jsx

```diff
@@ -71,28 +80,43 @@ class State extends React.Component {
       };
     });
   };
+  clearBurger = () => {
+    if (this.state.ingredientAddingOrder.length !== 0) {
+      const emptyBurgerCreator = {};
+      for (const ingredient in this.state.burgerCreator) {
+        emptyBurgerCreator[ingredient] = 0;
+      }
+      this.setState({
+        ingredientAddingOrder: [],
+        burgerCreator: emptyBurgerCreator,
+        orderPrice: "1.00",
+      });
+    }
+  };
   render() {
     const {
       prices,
       ingredients,
       orderPrice,
       burgerCreator,
       ingredientAddingOrder,
+      loading,
     } = this.state;
     return (
       <MainStyled>
         <Banner></Banner>
         <div className="Row">
-          <Prices priceList={prices} ingredients={ingredients}></Prices>
+          <Prices priceList={prices} loading={loading}></Prices>
           <Burger
             orderPrice={orderPrice}
             ingredientAddingOrder={ingredientAddingOrder}
           ></Burger>
           <Controls
-            priceList={prices}
             ingredients={ingredients}
             QuantitiesList={burgerCreator}
             updateBurger={this.handleChangeBurgerIngredientQuantity}
+            loading={loading}
+            clear={this.clearBurger}
```

Comment:
UI is jumping and not OK

https://user-images.githubusercontent.com/34886027/209449297-480db6e0-4b85-4721-9852-54f6b8e37731.mo

<img width="884" alt="Screenshot 2022-12-20 at 19 11 50" src="https://user-images.githubusercontent.com/34886027/209449310-694cf126-0bf7-4b1d-ab0e-756d8f3e7e69.png">
v

Reply:
https://user-images.githubusercontent.com/34886027/209833653-67d57bcc-eee0-4c56-b481-b15c64bcc876.mov

Reply:
Update UI. So it feels more user-friendly

Reply:
Burger making is not OK

https://user-images.githubusercontent.com/34886027/209833871-63dfc46c-6685-4cd5-94e3-d0876a1c1e46.mov